projects/kit/components/rating/rating.style.less

```diff
@@ -0,0 +1,79 @@
+:host {
+    position: relative;
+    display: inline-flex;
+
+    &._readonly,
+    &._disabled {
+        pointer-events: none;
+    }
+
+    &._disabled {
+        opacity: 0.5;
+    }
+}
+
+.t-rating {
+    &__wrapper,
```

Comment:
Please flatten the style sheet as per my comments to a different PR

Reply:
fixed